conf.py

```diff
@@ -293,3 +294,8 @@
 
 # Example configuration for intersphinx: refer to the Python standard library.
 intersphinx_mapping = {'https://docs.python.org/': None}
+
+extlinks = {
+    'oep-status': ('https://openedx.atlassian.net/browse/%s', 'OEP-')
```

Comment:
This seems to need to be:  `'oep-status': ('https://openedx.atlassian.net/browse/OEP-%s', 'OEP-')`

Also, I never used this before, I should put it in coverage.py docs...